dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -29,21 +30,24 @@ def funometer_by_script
       ratings = PuzzleRating.where('puzzle_ratings.script_id = ?', @script_id)
       @overall_percentage = get_percentage_positive(ratings)
 
-      # Generate the funometer percentages for the script, by day, for the last month.
-      @ratings_by_day_headers = ['Date', 'Percentage', 'Count']
-      @ratings_by_day, percentages_by_day = get_ratings_by_day(ratings)
+      # Generate the funometer percentages for the script, by day.
+      @ratings_by_day = get_ratings_by_day(ratings)
```

Comment:
hmm, so does this mean that the content of "ratings_by_day" is sometimes generated by the cron and sometimes by this helper method?